spec/integration/alerts_spec.rb

```diff
@@ -463,6 +475,17 @@ module Hawkular::Alerts::RSpec
       expect(alerts.size).to be >= 2
     end
 
+    it 'Should list alerts for multiple tenants' do
+      client = Hawkular::Alerts::Client.new(ALERTS_BASE, creds, options)
+
+      alerts = client.alerts(tenants: [:hawkular, :foo])
+
+      expect(alerts).to_not be_nil
+      expect(alerts.size).to be >= 4
+      expect(alerts.count { |a| a.tenantId == 'hawkular' }).to be >= 2
+      expect(alerts.count { |a| a.tenantId == 'foo' }).to be >= 2
```

Comment:
Can you put something like
```ruby
expect(alerts.count { |a| a.tenantId == 'hawkular' } + alerts.count { |a| a.tenantId == 'foo' }).to eq alerts.size
```
Just to be sure that we don't have alerts from any other tenant.